install/jsonb/bin/ts.jte

```diff
@@ -63,11 +63,15 @@ jsonb.classes=
 # @ts.classpath         --  Classes used to build the JSON-B tests
 # @ts.run.classpath     --  Classes needed by test run
 ####################################################################
+glassfish.modules.path=/home/jbescos/programs/glassfish6/glassfish/modules
+missing.jars=${ts.home}/lib
+ts.required.lib=${pathsep}${glassfish.modules.path}/yasson.jar${pathsep}${missing.jars}/jakarta.inject-api-2.0.0-RC4.jar${pathsep}${glassfish.modules.path}/jakarta.json.jar${pathsep}${glassfish.modules.path}/jakarta.json.bind-api.jar${pathsep}${glassfish.modules.path}/jakarta.servlet-api.jar
```

Comment:
how is jakarta.inject-api-2.0.0-RC4.jar different from ${glassfish.modules.path}/jakarta.inject-api.jar ?